src/Raven.Server/Documents/Handlers/RachisDatabaseHandler.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Threading.Tasks;
+using Raven.Server.Json;
+using Raven.Server.Routing;
+using Raven.Server.ServerWide.Context;
+
+namespace Raven.Server.Documents.Handlers
+{
+    public class RachisDatabaseHandler : DatabaseRequestHandler
+    {
+        [RavenAction("/databases/*/admin/rachis/wait-for-raft-commands", "POST", AuthorizationStatus.Operator)]
```

Comment:
Why Operator? DatabaseAdmin?

Reply:
@karmeli87 is this OK? this is the only database / admin / rachis endpoint

Reply:
Because this is used only for indexes (which you need to be an operator to deploy)